join.spec.md

```diff
@@ -131,48 +131,42 @@ Documents MUST define a {join__Graph} enum. Each enum value defines a subgraph.
 
 The {join__Graph} enum is used as input to {@join} directives which link fields and types to subgraphs.
 
-# Scalars
-
-Documents MUST include definitions of any scalars they use. Producers SHOULD include the `@specifiedBy` directive, but it is not mandatory.
-
-##! join__Never
+# Directives
 
-:::[definition](photos.graphql#join__Never)
+##! @join__graph
 
-The {join__Never} scalar is used to indicate the type of a field which **cannot be resolved**. This is typically due to a composition error. Producers SHOULD provide more specific error information with an {@error} directive on the relevant field.
+Declare subgraph metadata on {join__Graph} enum values. 
```

Comment:
I think we need to document the arguments.

eg, is "name" basically just an arbitrary string with fewer constraints on its syntax than the enum value that can be used by consumers to identity the graph? but it doesn't necessarily have to relate to any name in any other part of the universe (eg, it will be the subgraph name in managed federation, but doesn't have to be)?

and `url` is an endpoint that resolves GraphQL queries for the subgraph?

We should document that `name` values need to be unique across the values in this enum.

Reply:
Fixed on my branch although @trevor-scheer may want to read the questions above carefully still